fundingmanager_test.go

```diff
@@ -520,24 +520,63 @@ func openChannel(t *testing.T, alice, bob *testNode, localFundingAmt,
 	return fundingOutPoint
 }
 
-func assertMarkedOpen(t *testing.T, alice, bob *testNode,
-	fundingOutPoint *wire.OutPoint) {
-	state, _, err := alice.fundingMgr.getChannelOpeningState(fundingOutPoint)
-	if err != nil {
-		t.Fatalf("unable to get channel state: %v", err)
+func assertNumPendingChannels(t *testing.T, node *testNode, expectedNum int) {
+	var numPendingChans int
+	for i := 0; i < 10; i++ {
+		pendingChannels, err := node.fundingMgr.cfg.Wallet.Cfg.Database.FetchPendingChannels()
```

Comment:
more than 80 chars?

Reply:
Yep, not sure what's the best way to split such a line, but did my best now :b

Reply:
awesome, looks good!